README.md

```diff
@@ -1,2 +1,43 @@
 # hof-behaviour-emailer
 HOF behaviour to send emails
+
+## Usage
+
+```js
+const EmailBehaviour = require('hof-behaviour-emailer');
+
+// configure email behaviour
+const emailer = EmailBehaviour({
+  transport: 'ses',
+  transportOptions: {
+    accessKeyId: '...',
+    secretAccessKey: '...'
+  },
+  template: path.resolve(__dirname, './views/emails/confirm.html'),
+  from: 'confirmation@homeoffice.gov.uk',
+  recipient: 'customer-email',
+  subject: 'Application Successful'
+});
+
+// in steps config
+steps: {
+  ...
+  '/confirm': {
+    behaviours: ['complete', emailer],
```

Comment:
My understanding from @joefitter is that they are composed left to right, which means that later behaviours compose on top of earlier behaviours, and so end up higher in the composition stack and therefore execute first.

In this *particular* case, it doesn't actually matter, since this behaviour modifies `saveValues` and the `complete` behaviour extends `successHandler` so it'd work the same in either order.